ipaserver/install/ipa_backup.py

```diff
@@ -365,6 +366,11 @@ def add_instance_specific_data(self):
             if os.path.exists(file):
                 self.files.append(file)
 
+        self.files.append(
+            os.path.join(paths.IPA_PASSWD_DIR,
+                         HTTPD_PASSWD_FILE_FMT.format(host=api.env.host))
```

Comment:
I wonder, why did you add ``HTTPD_PASSWD_FILE_FMT`` to ``ipaplatform.constants`` instead of ``ipaplatform.paths`` as absolute path?

Reply:
It changes with the virtual host name, that's how the password reader works.

Reply:
I mean, why did you decide to split the directory and file name into two? Instead of
```
os.path.join(paths.IPA_PASSWD_DIR,
             HTTPD_PASSWD_FILE_FMT.format(host=api.env.host))
```

why not simply
```
paths.HTTPD_PASSWD_FILE_FMT.format(host=api.env.host)
```
?

Reply:
Hm, right, that's probably easier. I suppose I was thinking the IPA_PASSWD_DIR could be useful once but I can merge it to constants instead.